stor/tests/test_dx.py

```diff
@@ -1386,6 +1502,17 @@ def test_dx_canonical_to_dx_file(self):
 
 
 class TestCopyTree(DXTestCase):
+    def test_empty_path(self):
+        self.setup_temporary_project()
+        dx_p = DXPath('dx://')
+        dest = DXPath('dx://' + self.project)
+        with pytest.raises(exceptions.NotFoundError, match='No project or directory was found'):
```

Comment:
to me, this error message reads as if the destination doesn't *exist*.  Instead what you're trying to say is more like:

```
Cannot copy to or from root (dx://). Must specify project and/or directory or file.
```

Reply:
Fixed.